src/guide/typescript/overview.md

```diff
@@ -34,7 +34,7 @@ Volar sustituye a [Vetur](https://marketplace.visualstudio.com/items?itemName=oc
 
 - [TypeScript Vue Plugin](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) también es necesario para obtener soporte de tipo para las importaciones `*.vue` en los archivos TS.
 
-- [WebStorm](https://www.jetbrains.com/webstorm/) también proporciona soporte inmediato para TypeScript y Vue. Otros IDEs de JetBrains también los soportan, ya sea de forma inmediata o a través de [un plugin gratuito](https://plugins.jetbrains.com/plugin/9442-vue-js).
+- [WebStorm](https://www.jetbrains.com/webstorm/) también proporciona soporte inmediato para TypeScript y Vue. Otros IDEs de JetBrains también los soportan, ya sea de forma inmediata o a través de [un plugin gratuito](https://plugins.jetbrains.com/plugin/9442-vue-js). A partir de la versión 2023.2, WebStorm y el plugin Vue vienen con soporte integrado para el servidor de lenguaje Vue. Puedes configurar el servicio Vue para que utilice la integración Volar en todas las versiones de TypeScript, en Ajustes > Lenguajes y Frameworks > TypeScript > Vue. Por defecto, Volar se utilizará para las versiones de TypeScript 5.0 y superiores.
```

Comment:
"WebStorm y el plugin **de** Vue vienen con soporte integrado para el servidor de lenguaje **de** Vue." Igual para el resto del parrafo.

¿Qué opinan? @icarusgk @glopzel

Reply:
Pienso que a pesar de que es "Vue plugin" y "Vue Service", si se daría a entender mejor si usamos "El plugin **de** Vue", "El servicio **de** Vue" and so on...